infrastructure/models.py

```diff
@@ -400,6 +433,31 @@ def fuzzy_planned_completion_date(self):
             self.planned_completion_month,
             self.planned_completion_day
         )
+
+    @property
+    def pipeline_capacity_property(self):
```

Comment:
These ``_property`` functions feel like template display logic to me. Is there a reason why this is in the model and not in the template?

Reply:
To me it seemed just a little more elegant, I'm more than happy to move this into the template though.

Reply:
Based on slack convo. I'm going to leave this as is.